smart-contracts/assembly/contracts/FT/WMAS.ts

```diff
@@ -33,9 +50,16 @@ export function withdraw(bs: StaticArray<u8>): void {
   const recipient = new Address(
     args.nextString().expect('recipient is missing'),
   );
-
-  assert(amount > 0, 'Payment must be more than 0 WMAS');
-
   burn(u256ToBytes(u256.fromU64(amount)));
   transferCoins(recipient, amount);
 }
+
+export function computeStorageCost(receiver: Address): u64 {
```

Comment:
it's a storage cost for a mint, maybe specify in the function name because it would be different for a increaseAllowance for example